src/emmc.c

```diff
@@ -771,6 +773,11 @@ pu_emmc_parse_partitions(PuEmmc *emmc,
 
     emmc->partitions = g_list_reverse(emmc->partitions);
 
+    /* Reserve 34 blocks for secondary GPT */
+    if (g_str_equal(emmc->disktype->name, "gpt")) {
+        fixed_parts_size += SECONDARY_GPT_SECTOR_COUNT;
```

Comment:
It is a bit hacky to increment `fixed_parts_size` as the blocks reserved for the secondary partition table are not related to the fixed partitions total size. We should probably decrement `expanded_part_size` instead.